scripts/tasks/jest.js

```diff
@@ -15,7 +15,7 @@ module.exports = function (options) {
       `--config ${jestConfigPath}`,
 
       // Run tests in serial (parallel builds seem to hang rush.)
-      `--runInBand`,
+      process.env.TRAVIS ? `--runInBand` : undefined,
```

Comment:
The comment probably needs to be updated

Reply:
Also, is there a reason to exempt the CI build from parallel jest?

Reply:
One other issue is that TRAVIS isn't the only "CI" entity - some builds take place on VSTS (production/hotfix). 

Something we need to fix all-up (might be out of scope for this PR)